stats/cloud/errors.go

```diff
@@ -38,26 +38,65 @@ var (
 type ErrorResponse struct {
 	Response *http.Response `json:"-"`
 
-	Code    int               `json:"code"`
-	Message string            `json:"message"`
-	Details map[string]string `json:"details"`
+	Code        int                 `json:"code"`
+	Message     string              `json:"message"`
+	Details     map[string][]string `json:"details"`
```

Comment:
I'm a bit confused. If this field changed from `map[string]string` to `map[string][]string`, how will this work for both old and new k6 versions? Won't trying to decode it result in an error in the old k6 versions?

Reply:
I am pretty sure this is what new 72-74 lines are for (can't link :()

Reply:
What I meant was that the `json.Unmarshal()` call itself will likely fail in the old version with the change of type for `details`. It's fine to add new fields in the API response, but changing the type of an existing field seems problematic for the older k6 versions...

Reply:
Sorry, missed that in the PR description... So the backend response change of `details` from `map[string]string` to `map[string][]string` happened a long time ago, not as a part of the recent refactoring, and this just fixes the bug?

Reply:
The recent refactoring has not changed the format, but I discovered the issue when testing it. It may have never worked or was changed long time ago. 🤦‍♂️

This just fixes the bug in `k6` to handle it properly.